PathPlanning/DynamicWindowApproach/dynamic_window_approach.py

```diff
@@ -48,9 +47,10 @@ def __init__(self):
         self.yaw_rate_resolution = 0.1 * math.pi / 180.0  # [rad/s]
         self.dt = 0.1  # [s] Time tick for motion prediction
         self.predict_time = 3.0  # [s]
-        self.to_goal_cost_gain = 0.15
+        self.to_goal_cost_gain = 0.2
```

Comment:
Could you please keep previous parameter as default value and change parameter in the test function?